### PR TITLE
New package: pcsc-cyberjack-3.99.5

### PR DESCRIPTION
**Pull request by dataCobra**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64)
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - aarch64-musl (crossbuild)
  - x86_64-musl (crossbuild)
  - i686-musl (crossbuild)
  - i686 (crossbuild)
  - armv7l (crossbuild)
  - armv6l-musl (crossbuild)

### CODE REVIEW

**Comment by dataCobra** [comment]
I've fully tested the PR on my system. It works flawless. :+1:

**Comment by dataCobra** [comment]
@classabbyamp or @Duncaen could you help me with the check error message?

I don't know why this error appears.

Building on my machine worked without a problem.

**Comment by paper42** [comment]
> could you help me with the check error message?
> 
> I don't know why this error appears.
> 
> Building on my machine worked without a problem.

CI is currently broken, you can ignore it, but please don't ping random people.

**Comment by dataCobra** [comment]
The checks passed. +1

I've successful tested this package in combination with PR #39063.

**Comment by dataCobra** [comment]
Hey @classabbyamp,

you added the label so I assume you're the reviewer for this PR?

I use this now several weeks without any issues.

I think it can be merged. :+1:

**Comment by classabbyamp** [comment]
no I added the label because I saw it in the feed on IRC...

File: srcpkgs/pcsc-cyberjack/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+pkgname=pcsc-cyberjack
+version=3.99.5
+revision=1
+_versext=SP15
+wrksrc=pcsc-cyberjack-${version}final.${_versext}
+build_style=gnu-configure
+configure_args="--enable-udev"
+hostmakedepends="pkg-config"
+makedepends="pcsclite-devel eudev-libudev-devel libusb-compat-devel"
+depends="pcsc-ccid libusb"
```
Comment (classabbyamp) [inline srcpkgs/pcsc-cyberjack/template:11]: `libusb` shouldn't be necessary, it's detected: `SONAME: libusb-1.0.so.0 <-> libusb>=1.0.0_1`

File: srcpkgs/pcsc-cyberjack/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+short_desc="Linux drivers for cyberJack® RFID"
+maintainer="dataCobra <datacobra@thinkbot.de>"
+license="LGPL-2.1-or-later, GPL-2.0-or-later"
+homepage="https://www.reiner-sct.com/lang/en/support"
+#changelog=""
```
Comment (classabbyamp) [inline srcpkgs/pcsc-cyberjack/template:16]:
```suggestion
```

Comment (dataCobra) [reply srcpkgs/pcsc-cyberjack/template]: I checked once more for a changelog but couldn't find one. I removed this line as suggested. :+1:

File: srcpkgs/pcsc-cyberjack/template (hunk 1 above)
Comment (dataCobra) [reply srcpkgs/pcsc-cyberjack/template]: I've removed the libusb dependency and it build without a problem. Thanks for your hint. :+1:

**Comment by dataCobra** [comment]
It's a month now where I use the packages without a problem.

Do I miss something to get this merged?

The devices are officially supported by AusweisApp2 and also encouraged by the company Governikus.

**Pull request closed without merging**